packages/web-components/src/components/ic-input-label/ic-input-label.tsx

```diff
@@ -62,6 +64,27 @@ export class InputLabel {
     );
   }
 
+  private isHelperTextSlotUsed = (slot: Element | null): boolean => {
+    const assignedEls = (slot as HTMLSlotElement)?.assignedElements();
```

Comment:
suggestion to help with coverage: could you create getAssignedElements function which you could then mock in the tests

eg 
```
private getAssignedElements (slot: Element) => {
 return (slot as HTMLSlotElement)?.assignedElements();
}
```

Reply:
Managed to get something working before I saw your comment, but it's similar - I've just mocked assignedElements() directly

I tried to test the recursion part of isHelperTextSlot used but it says those lines are still uncovered, so it might not be possible